.coveragerc

```diff
@@ -1,4 +1,7 @@
 [run]
+source = pyiddidf
```

Comment:
Point the coverage binary to focus on our specific library, and move the omit keyword to the report section.